float64[] red_center
float64[] green_center
float64[] blue_center